inscriptions/ISic003323.xml

```diff
@@ -199,12 +199,12 @@
             <div type="edition" xml:space="preserve" xml:lang="la" resp="#DEE">
                 <ab>
                     <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/><persName type="attested"><name type="nomen"><seg part="F"><unclear>i</unclear><unclear>o</unclear></seg></name> <name type="cognomen"><supplied reason="lost">I</supplied><unclear>n</unclear><unclear>n</unclear><unclear>o</unclear><unclear>c</unclear>enti</name></persName>
-                    <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/><gap reason="illegible" quantity="1" unit="character"/>V <gap reason="lost" extent="unknown" unit="character"/>
-                    <lb n="3"/><gap reason="lost" extent="unknown" unit="character"/><gap reason="illegible" quantity="2" unit="character"/><unclear>NIA</unclear><gap reason="illegible" quantity="1" unit="character"/>NIS <g ref="#interpunct">·</g> HV <gap reason="lost" extent="unknown" unit="character"/> <w>obito</w> <g ref="#interpunct">·</g> <w>ex</w>
+                    <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/><orig>v</orig> <gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="3"/><gap reason="lost" extent="unknown" unit="character"/><orig><unclear>t</unclear></orig><orig><unclear>nia</unclear></orig><orig><unclear>nis</unclear></orig> <g ref="#interpunct">·</g> <orig>hu</orig> <gap reason="lost" extent="unknown" unit="character"/> <w>obito</w> <g ref="#interpunct">·</g> <w>ex</w>
                     <lb n="4"/><w><supplied reason="lost">testamento</supplied></w> <w><supplied reason="lost">eius</supplied></w> <w><supplied reason="lost">os</supplied><unclear>s</unclear>a</w> <w>h<supplied reason="lost"><choice><corr>hic</corr><sic>eic</sic></choice></supplied></w> <w><supplied reason="lost">in</supplied></w> <w><supplied reason="lost">cella</supplied></w> <w><supplied reason="lost">absi</supplied><unclear>d</unclear>ata</w> <g ref="#interpunct">·</g> <w>sunt</w> <g ref="#interpunct">·</g>
```

Comment:
Remove the 'h' from `<corr>`, it's already there in the beginning of the word before the correction.